docker/dockerfile.merlin

```diff
@@ -10,6 +10,14 @@ FROM ${FULL_IMAGE} as triton
 FROM ${DLFW_IMAGE} as dlfw
 FROM ${BASE_IMAGE} as build
 
+FROM ${BASE_IMAGE} as build-amd64
+ONBUILD COPY --chown=1000:1000 --from=triton /opt/tritonserver/backends/fil /opt/tritonserver/backends/fil/
+
+FROM ${BASE_IMAGE} as build-arm64
+RUN echo "Skipping copy of /opt/tritonserver/backends/fil. (Why does this fail on arm64?)"
```

Comment:
Why are these calls done separately? The base image will be the same throughout the build right?

Reply:
They must be separated, not combined.  Lines 13-14 are for amd64 (i.e. x86_64) while lines 16-17 are for arm64 (i.e aarch64).  This is one way to do a conditional copy in Docker where different file paths are copied depending on the target architecture.